debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/WhatIsCommand.java

```diff
@@ -347,7 +347,7 @@ private boolean walkStructuresFrom(StructurePointer startPoint) throws DDRIntera
 			} catch (InvocationTargetException e) {
 				Throwable cause = e.getCause();
 				
-				if (cause instanceof CorruptDataException || cause instanceof NoSuchFieldError || cause instanceof NoClassDefFoundError) {
+				if (cause instanceof CorruptDataException || cause instanceof NoSuchFieldException || cause instanceof NoSuchFieldError || cause instanceof NoClassDefFoundError) {
```

Comment:
I'm not sure this should have been catching `NoClassDefFoundError` or `NoSuchFieldError`; please explain why we want to continuing catching them.